### ISSUE TITLE
Vertically oriented sprites

### ISSUE THREAD

**Issue by probablykabari** [aberant/css-spriter#1]
They would be cool.

**Comment by probablykabari** [comment]
I'm going to tackle this in my fork.

**Comment by tjennings** [comment]
yeah go ahead, it'd be nice to have.  Beyond it being cool, are their advantages to vertically oriented sprites?

**Comment by probablykabari** [comment]
Not really any _advantages_, just a preference depending on the images and what is being done with them. I see this as a step toward having grid-like sprites, which would be very useful.
